main.groovy

```diff
@@ -0,0 +1,349 @@
+/*
+* Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+*/
+import groovy.io.FileType
+import hudson.model.*
+
+def deploymentDirectories = []
+def updateType = ""
+def s3BucketName = "janeth-tg-logs"
+def s3BuildLogPath = ""
+
+pipeline {
+agent {label 'pipeline-slave'}
```

Comment:
Shall we rename the label to `pipeline-agent`?

Reply:
Fixed